abstract/group.go

```diff
@@ -51,6 +51,9 @@ type Secret interface {
 
 	// Set to a fresh random or pseudo-random secret
 	Pick(rand cipher.Stream) Secret
+
+	// SetBytes will take bytes and create a secret out of it
+	SetBytes([]byte) Secret
```

Comment:
Yes ok,  I get it this way. For me it still does not make much sense to for `SetBytes` and `Bytes` to not use the same slice. Example with `Un/Marshal` we can do :
```
a := sec.MarshalBinary()
sec.UnmarshalBinary(a)
a2 := sec.MarshalBinary()
a == a2
```
with `Bytes`:
```
secret.SetBytes(a)
a2 := secret.Bytes()
a != a2
```
But ok, if that makes sense for most of you then I abide :)